src/jquery.addressfield.js

```diff
@@ -65,6 +65,9 @@
         url: configs.json,
         async: configs.async,
         success: function (data) {
+          if($(configs.fields.country)[0].children.length === 0){
```

Comment:
Perhaps its best to keep all the logic with the selector itself?  Like...
```javascript
if ($(configs.fields.country + ':not(:has(>option))') {/*...*/}
```

I might also recommend you encapsulate this logic within your `initCountries` method, though.  We should be calling this both in the case of the AJAX JSON call as well as the inline configuration object in the `else if` just below this.